demo/src/main/java/com/example/demo/repo/VideoRepository.java

```diff
@@ -0,0 +1,43 @@
+package com.example.demo.repo;
+
+import com.example.demo.models.UserDataBase;
+import com.example.demo.models.Video;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public interface VideoRepository extends JpaRepository<Video, Long> {
+    @Transactional
+    public Video findVideoByName(String nameVideo);
```

Comment:
Is it really used anywhere? Name is not unique, and searching by name is a more complicated thing anyway. Better to remove this for now and discuss later (see SQL LIKE operator)